SU-SWS/stanford_profile
# Fixes selector strength between stanford page and stanford page full.

# READY FOR REVIEW 

# Summary
- Adds modular margin to bottom of top banner
- Moves full page scss to full page scss file

# Needed By (Date)
- Today

# Urgency
- High

# Steps to Test

1. Check out this branch
2. Clear all caches
3. Review the home page margin on the top banner and make sure it is there
4. Resize the browser window and ensure the paragraph types don't touch the edge that aren't banners.

# Affected Projects or Products
- D8CORE

# Associated Issues and/or People
- D8CORE

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
